connectors/src/connectors/microsoft/temporal/activities.ts

```diff
@@ -508,8 +529,13 @@ export async function syncOneFile({
 async function handlePptxFile(
   data: ArrayBuffer,
   fileId: string | undefined,
-  localLogger: Logger
+  localLogger: Logger,
+  maxDocumentLen: number
 ): Promise<CoreAPIDataSourceDocumentSection | null> {
+  if (data.byteLength > 4 * maxDocumentLen) {
```

Comment:
where does the 4 come from? is that the same as what's done in gdrive? if so, would that make sense to push it in a common lib?

Reply:
Comes from GDrive indeed. I'm planning to do a refactoring and have GDrive and Microsoft share a bunch of resources.
Will do that later as there's a GDrive refacto PR ongoing and Flavien is also adding support for Apache Tika